.github/workflows/main.yml

```diff
@@ -24,27 +24,27 @@ jobs:
               run: yarn run build
             - name: lint
               run: yarn run lint
- 
```

Comment:
Undo the indent changes.

Reply:
These indents are added precommit hook of husky. I think that i cannot stop those.